tests/test_v2_error.py

```diff
@@ -55,47 +55,47 @@ def test_raises_v2_error(
             api_key="keyinfo_test_123",
         )
 
-    @pytest.mark.skip("python doesn't have any errors with invalid params yet")
-    def test_raises_v2_error_with_field(
-        self,
-        stripe_client: stripe.StripeClient,
-        http_client_mock: HTTPClientMock,
-    ):
-        method = "post"
-        path = "/v2/payment_methods/us_bank_accounts"
-
-        error_response = {
-            "error": {
-                "type": "invalid_payment_method",
-                "code": "invalid_us_bank_account",
-                "message": "bank account is invalid",
-                "invalid_param": "routing_number",
-            }
-        }
-        http_client_mock.stub_request(
-            method,
-            path=path,
-            rbody=json.dumps(error_response),
-            rcode=400,
-            rheaders={},
-        )
-
-        try:
-            stripe_client.v2.payment_methods.us_bank_accounts.create(
-                params={"account_number": "123", "routing_number": "456"}
-            )
-        except error.InvalidPaymentMethodError as e:
-            assert e.invalid_param == "routing_number"
-            assert e.error.code == "invalid_us_bank_account"
-            assert e.error.message == "bank account is invalid"
-        else:
-            assert False, "Should have raised a InvalidUsBankAccountError"
-
-        http_client_mock.assert_requested(
-            method,
-            path=path,
-            api_key="keyinfo_test_123",
-        )
+    # @pytest.mark.skip("python doesn't have any errors with invalid params yet")
```

Comment:
this is pre-emptive test for a class of errors that don't exist yet. It was giving linting errors now that our imports were better, so I commented it out